### PR TITLE
Update to 0.7.0.

### PR DESCRIPTION
**Pull request by abergeron**

### CODE REVIEW

**Comment by abergeron** [comment]
I forgot to wait for 0.7.0 of libgpuarray.  This means that all the builders will fail for now.

**Comment by jakirkham** [comment]
No worries. Can easily restart the CIs later.

**Comment by jakirkham** [comment]
It would be worthwhile to [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) this with `conda-smithy` 2.3.3 as well.

**Comment by abergeron** [comment]
Done

**Comment by jakirkham** [comment]
Thanks. LGTM.

At some point we should adopt the new way of handling NumPy, but there is no need for that now.

**Comment by abergeron** [comment]
There might be a 0.7.1 soon-ish so if you have some documentation on what is the new way to handle NumPy I can roll that in.

**Comment by jakirkham** [comment]
There are some [docs]( https://conda-forge.org/docs/meta.html#building-against-numpy ), but the version constraints are not totally settled yet (Windows in particular). So IMHO would wait to adopt it until that is resolved.

**Comment by abergeron** [comment]
Well I guess I'll have to address it now since I get an error due to "numpy x.x" with AppVeyor.

What do you recommend?

File: appveyor.yml [hunk 1]
```diff
@@ -10,95 +10,13 @@ environment:
...
-
-    - TARGET_ARCH: x86
-      CONDA_NPY: 113
-      CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda36
-
-    - TARGET_ARCH: x64
-      CONDA_NPY: 113
-      CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda36-x64
```
Comment (jakirkham) [inline appveyor.yml]: I think the issue on AppVeyor is because this matrix got removed. It shouldn't have been. 😕 Could you please try re-rendering again to see if the issue persists?

Comment (jakirkham) [reply appveyor.yml]: If that doesn't work, can just revert the changes on this file manually. The re-rendering should have left it unchanged FWICT.

Comment (jakirkham) [reply appveyor.yml]: Ah! I think this is failing because `libgpuarray` 0.7.0 for Windows was not published yet. Once that happens re-rendering should keep the matrix.

Note: Sorry for so many comments.

File: .travis.yml [hunk 2]
```diff
@@ -7,17 +7,6 @@ os: osx
...
-    
-    - CONDA_NPY=111  CONDA_PY=27
-    - CONDA_NPY=112  CONDA_PY=27
-    - CONDA_NPY=113  CONDA_PY=27
-    - CONDA_NPY=111  CONDA_PY=35
-    - CONDA_NPY=112  CONDA_PY=35
-    - CONDA_NPY=113  CONDA_PY=35
-    - CONDA_NPY=111  CONDA_PY=36
-    - CONDA_NPY=112  CONDA_PY=36
-    - CONDA_NPY=113  CONDA_PY=36
```
Comment (jakirkham) [inline .travis.yml]: Looks like we have the same issue here. Would just manually revert the changes to `.travis.yml` and `appveyor.yml`. They are both caused by `libgpuarray` 0.7.0 missing packages for macOS and Windows respectively.

File: appveyor.yml (hunk 1 above)
Comment (abergeron) [reply appveyor.yml]: I think you're right.  I just did a re-render and the matrix was back.

I amended the render commit to have a clean history.

File: .travis.yml (hunk 2 above)
Comment (abergeron) [reply .travis.yml]: I just did that
